nix/ghc9.cabal.project

```diff
@@ -29,15 +35,9 @@ source-repository-package
 source-repository-package
     type: git
     location: https://github.com/fpco/inline-c
-    --sha256: 0hi4s3p0whf0ndsxn8qrfc3a7qndsgac0vz7n23kp3rx69yp0yr1
-    tag: 2d0fe9b2f0aa0e1aefc7bfed95a501e59486afb0
```

Comment:
This commit got removed from the repo... so I updated it to the latest commit, and updated hasktorch as well.
![image](https://github.com/plow-technologies/inferno/assets/10712637/8cbbd81f-a0ce-4dd7-bd5f-1efa36c882d6)